### ISSUE TITLE
Postfix & suffix problems

### ISSUE THREAD

**Issue by lenartovich-17** [glennreyes/react-countup#326]
useCountUp hook does not support postfix and prefix settings. No effect when you use them.

**Comment by mmarkelov** [comment]
@Al2BeO4 could you provide example. It should be ok:
```js
import React from 'react';
import {useCountUp} from "react-countup";

function App() {
  const { countUp } =  useCountUp({ duration: 2, end: 10000, prefix: 'a', suffix: '!' });

  return (
    <div className="App">
      <header className="App-header">
        <text>{countUp}</text>
      </header>
    </div>
  );
}

export default App;

```

**Comment by Al2BeO4** [comment]
It`s ok! But...

prefix - I have these variables (user input), i.e. when the user enters a different value, useCountUp feeds the old object with the old settings. How do I make a "hardreset" so that a new object with new settings returns?

**Comment by mmarkelov** [comment]
@Al2BeO4 yeah! Some research on it. There is some difference with component and hook versions: 
We just recreate countup with component, but we do not do it for hook version, I suppose we should be consistence and hook version should also recreate countup instance.
I'll add this logic in #321

**Comment by mmarkelov** [comment]
should be fixed in **react-countup@5.0.0-0**
